src/handlers/QuotingHandler.js

```diff
@@ -27,9 +29,7 @@ class QuotingHandler {
       throw reformatFSPIOPError(error)
     }
 
-    await Promise.allSettled(
-      messages.map(msg => this.defineHandlerByTopic(msg))
-    )
+    messages.map(msg => this.defineHandlerByTopic(msg))
```

Comment:
Could you explain, please, what's the point to acknowledge all messages right after consuming it without awaiting actual processing?